cms/workflows/models.py

```diff
@@ -22,6 +25,17 @@ class Meta:
 class ReadyToPublishGroupTask(AbstractGroupApprovalTask):
     """Placeholder task model to use in the Bundle approval logic."""
 
+    def locked_for_user(self, obj: "Model", user: "User") -> bool:
+        # TODO: update lock message (needs PR to core)
```

Comment:
Is there like an issue you raised, Dan, that exists, and is JIRA card tracking this? We have been tracking Wagtail core issues raised with a simple JIRA card just for awareness.

Potentially, in the future, we may have a confluence page pulling in these JIRA cards to track Wagtail core issues.

Reply:
CMS-473 / https://github.com/wagtail/wagtail/issues/13075

Reply:
Thanks, Dan, for creating these. Good stuff! Resolving!